tools/editor/multi_node_edit.cpp

```diff
@@ -167,6 +181,11 @@ void MultiNodeEdit::add_node(const NodePath& p_node){
 	nodes.push_back(p_node);
 }
 
+void MultiNodeEdit::set_property_field(const StringName& p_property, const Variant& p_value, const String& p_field) {
+
+	_set_impl(p_property, p_value, p_field);
```

Comment:
You can probably move all of `_set_impl` here, and then call `set_property_field(p_name, p_value, "")` in `_set`.